core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -543,26 +546,29 @@ object Boot extends IOApp with LazyLogging {
     val pathToCredentialJson = gcsConfig.getString("pathToCredentialJson")
     val jsonFileSource = scala.io.Source.fromFile(pathToCredentialJson)
     val jsonCreds = try jsonFileSource.mkString finally jsonFileSource.close()
+    val saCredentials = ServiceAccountCredentials.fromStream(new ByteArrayInputStream(jsonCreds.getBytes(StandardCharsets.UTF_8)))
```

Comment:
Out of curiosity, what is the reason to converting this to a byte array before reading it in? Does `ServiceAccountCredentials` have a from JSON method?

Reply:
Sadly not - or not one that I could find. Would be nice though. The main problem here is that we're reading in this file many times and it's ugly. Maybe should create a todo to unify this.

Reply:
https://broadworkbench.atlassian.net/browse/CA-1806